app/views/puppetclasses/_form.html.erb

```diff
@@ -1,5 +1,8 @@
 <%= form_for @puppetclass, :url => (@puppetclass.new_record? ? puppetclasses_path : puppetclass_path(:id => @puppetclass)) do |f| %>
   <%= base_errors_for @puppetclass %>
+  <% unless @puppetclass.errors.empty? %>
+    <%= alert :class => 'alert-danger', :header => '', :text => _("The class has not been saved because of an error in the class or one of its parameters.") %>
```

Comment:
could you reword this to `The class could not be saved because of an error in one of the class parameters.`?